File: exercises/beecrowd/1158/1158.go
// https://judge.beecrowd.com/pt/problems/view/1158
package main

import "fmt"

func main() {
	var n int
	fmt.Scan(&n)

	for i := 0; i < n; i++ {
		var x, y int
		fmt.Scan(&x, &y)

		if x%2 == 0 {
			x++
		}

		sum := 0
		for j := 0; j < y; j++ {
			sum += x + 2*j
		}

		fmt.Println(sum)
	}
}

File: exercises/beecrowd/1158/1158.java
// https://judge.beecrowd.com/pt/problems/view/1158
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();
        while (n-- > 0) {
            int x = sc.nextInt();
            int y = sc.nextInt();
            int sum = 0;

            while (y > 0) {
                if (x % 2 != 0) {
                    sum += x;
                    y--;
                }
                x++;
            }

            System.out.println(sum);
        }
        sc.close();
    }
}
